engine/index.js

```diff
@@ -462,3 +462,8 @@ export default {
   buildStatus,
   buildIndex,
 };
+
+const test = {
```

Comment:
It turns out that rewire doesn't work with babel: https://github.com/jhnns/rewire/issues/62

However there is a babel plugin that might work for this use case: https://github.com/speedskater/babel-plugin-rewire

I'm giving it a shot now